config.go

```diff
@@ -33,6 +33,17 @@ type Process struct {
 	Cwd string `json:"cwd"`
 }
 
+// User specifies linux specific user and group information for the container's
+// main process.
+type User struct {
+	// UID is the user ID the Process is executed as. (this field is platform dependent)
```

Comment:
Good idea

On Wed, Jan 13, 2016, 15:57 Vish Kannan <notifications@github.com> wrote:

> In config.go
> <https://github.com/opencontainers/specs/pull/310#discussion_r49670377>:
>
> > @@ -33,6 +33,17 @@ type Process struct {
> >  	Cwd string `json:"cwd"`
> >  }
> >
> > +// User specifies linux specific user and group information for the container's
> > +// main process.
> > +type User struct {
> > +	// UID is the user ID the Process is executed as. (this field is platform dependent)
>
> Should we use a json tag to indicate platform dependency? It will help
> with tooling as well.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/opencontainers/specs/pull/310/files#r49670377>.
>